mongodbatlas/resource_mongodbatlas_cluster_test.go

```diff
@@ -486,21 +484,20 @@ func testAccCheckMongoDBAtlasClusterDestroy(s *terraform.State) error {
 func testAccMongoDBAtlasClusterConfigAWS(projectID, name, backupEnabled string) string {
 	return fmt.Sprintf(`
 		resource "mongodbatlas_cluster" "test" {
-			project_id   = "%[1]s"
-			name         = "%[2]s"
-			disk_size_gb = 100
-			num_shards   = 1
+			project_id                   = "%[1]s"
+			name                         = "%[2]s"
+			disk_size_gb                 = 10
+			num_shards                   = 1
 			replication_factor           = 3
 			provider_backup_enabled      = %[3]s
-			pit_enabled 				 = %[3]s
```

Comment:
why would we remove pit_enabled?  We want to test this as it's one of the most needed things in 4.0 as it's a new flag for those using provider_backup_enabled to have PIT restores turned on.

Reply:
I agree with @themantissa, is there any reason why you (@PacoDw ) removed?